go/iot/protocol/Protocol.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protocol
+
+const (
+	OK                = 200
+	FAIL              = 404
+	UNREGISTERTAG     = 501
+	UNREGISTERMEASURE = 502
+)
+
+type MeasureArgument struct {
+	Name         string
+	Measurement string
+	Tag          map[string]interface{}
```

Comment:
Why `Tag` is a `map[string]interface{}` instead of `map[string]string`, given that we only support string as `TagValue`?

Reply:
Also consider renaming this to `Tags`.

Reply:
your are right. I am using the assertion in the later tag insertion which is useless.